azure-devops/azext_devops/dev/boards/work_item.py

```diff
@@ -42,6 +43,8 @@ def create_work_item(work_item_type, title, description=None, assigned_to=None,
     :param discussion: Comment to add to a discussion in a work item.
     :type discussion: str
     :param fields: Space separated "field=value" pairs for custom fields you would like to set.
+    Refer https://docs.microsoft.com/en-us/rest/api/azure/devops/wit/fields/list?view=azure-devops-rest-5.0
```

Comment:
Do not share direct links. You will need an aka.ms link. 
@geverghe generally creates them so there is a single owner.

Reply:
https://aka.ms/azure-devops-cli-field-api - let me know if the url needs to be edited

Reply:
This looks fine.